### PR TITLE
Introduce `Rack::VERSION`.

### PR DESCRIPTION
**Pull request by ioquatix**

### CODE REVIEW

**Comment by Earlopain** [comment]
Is it worth to deprecate `Rack::RELEASE` already? It would be nice to keep both around for a while to give time for applications to drop older rack versions so they don't also have to case-when around this in addition to everything else.

**Comment by ioquatix** [comment]
I'm okay with considering different options (e.g. do nothing, or not deprecating it), however it will be between 6-12 months before this is actually released, and then there will be about 6-12 months for another release which actually removes it.

**Review by jeremyevans — CHANGES_REQUESTED** [review]
I agree with @Earlopain , that we don't need to deprecate `RELEASE`.

File: test/spec_version.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@
 describe Rack do
   describe 'version' do
     it 'is a version string' do
-      Rack::RELEASE.must_match(/\d+\.\d+\.\d+/)
+      Rack::VERSION.must_match(/\d+\.\d+\.\d+/)
```
Comment (jeremyevans) [inline test/spec_version.rb:12]: Please keep both constants here.  I would also add `Rack.release`

**Review by jeremyevans — CHANGES_REQUESTED** [review]
One small change requested, then this should be good to merge.

File: test/spec_version.rb (hunk 1 above)
Comment (ioquatix) [reply test/spec_version.rb:12]: Good idea.

**Review by jeremyevans — APPROVED** [review]

**Pull request merged**